flink-catalog-examples/README.md

```diff
@@ -0,0 +1,271 @@
+           ██░ ▓█▒█▒  ▒▓▓▒  ▓█                █░      ░░░░   ░█▒
+           ▓█   ▒█▓   ░     █░                ▒█              █▓
+            █▓   ██         █░                 ▓▓        ▒█▓▓▓▒█░
+             █▓ ░▓██░       ▓▒                  ▓█▓▒░░░▒▓█░    ▒█
+              ██   ▓█▓░      ▒                    ░▒█▒██▒      ▓▓
+               ▓█▒   ▒█▓▒░                         ▒▒ █▒█▓▒▒░░▒██
+                ░██▒    ▒▓▓▒                     ▓██▓▒█▒ ░▓▓▓▓▒█▓
+                  ░▓██▒                          ▓░  ▒█▓█  ░░▒▒▒
+                      ▒▓▓▓▓▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░▓▓  ▓░▒█░
+
+    ______ _ _       _       _____  ____  _         _____ _ _            _  BETA   
+   |  ____| (_)     | |     / ____|/ __ \| |       / ____| (_)          | |  
+   | |__  | |_ _ __ | | __ | (___ | |  | | |      | |    | |_  ___ _ __ | |_
+   |  __| | | | '_ \| |/ /  \___ \| |  | | |      | |    | | |/ _ \ '_ \| __|
+   | |    | | | | | |   <   ____) | |__| | |____  | |____| | |  __/ | | | |_
+   |_|    |_|_|_| |_|_|\_\ |_____/ \___\_\______|  \_____|_|_|\___|_| |_|\__|
+
+        Welcome! Enter 'HELP;' to list all available commands. 'QUIT;' to exit.
+
+
+Flink SQL>
+```
+
+After that, let's create a Pravega Catalog to access the data stored in Pravega.
+We run the following DDL statement in SQL CLI to create a Flink catalog that connects to Pravega and Schema Registry.
+
+```sql
+CREATE CATALOG pravega WITH(
+    'type' = 'pravega',
+    'controller-uri' = 'tcp://pravega:9090',
+    'schema-registry-uri' = 'http://schemaregistry:9092',
+    'default-database' = 'examples',
+    'serialization.format' = 'Avro'
+);
+
+USE CATALOG pravega;
+```
+
+The Pravega catalog will translate the schema from Schema Registry service and use it to create a flink table.
+Therefore, Pravega `scope` and `stream` will be treated as `database` and `table` in Flink so that we can access the data in Pravega.
+We can check the schema and query the data in Pravega catalog: 
+```sql
+DESCRIBE userBehavior;
+```
+![sql-describe](images/sql-describe.png)
+```sql
+SELECT * FROM userBehavior;
+```
+![query-all](images/query-all.gif)
+
+Now let's take a deeper look at two real-life scenarios to see how powerful Pravega catalog can be.
+
+### Use case 1: Unique Visitors (UV)
+
+One typical use case in the online services is to calculate the cumulative unique visitors continuously, i.e. every ten minutes in one day. 
+For example, the number of UV at 10:00 represents the total number of UV from 00:00 to 10:00.
+This can be easily and efficiently implemented by `CUMULATE` windowing.  
+You can check more about `CUMULATE` and other Windowing TVFs [here](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/queries/window-tvf/).
+
+In order to enrich the table with the event time and watermark information,
+we need to extend the existing catalog table like this using [LIKE clause](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/create/#like):
```

Comment:
With such enrichment, the new table inherits all the Pravega and schema information from the Pravega catalog table, thus, querying on the new table would work the same as we query the catalog table like the previous example.
Please add these words because the main idea is to introduce the power of the catalog.

Reply:
fixed